fdbclient/GlobalConfig.actor.h

```diff
@@ -77,23 +80,39 @@ class GlobalConfig : NonCopyable {
 	// database.
 	template <class T>
 	static void create(Database& cx, Reference<AsyncVar<T> const> db, const ClientDBInfo* dbInfo) {
-		if (g_network->global(INetwork::enGlobalConfig) == nullptr) {
+		auto config_map =
+		    reinterpret_cast<ConfigMap*>(g_network->global(INetwork::enGlobalConfig));
+		if (config_map == nullptr) {
+			auto m = new ConfigMap();
+			g_network->setGlobal(INetwork::enGlobalConfig, m);
+			config_map = m;
+		}
+
+		auto it = config_map->find(cx.dbId());
+		if (it == config_map->end()) {
 			auto config = new GlobalConfig{ cx };
 			g_network->setGlobal(INetwork::enGlobalConfig, config);
```

Comment:
This line is no longer needed. Instead, `config` should be inserted into `config_map`.